crates/compiler/roc_target/src/lib.rs

```diff
@@ -36,8 +48,8 @@ pub enum Architecture {
 impl std::fmt::Display for Architecture {
     fn fmt(&self, f: &mut std::fmt::Formatter) -> std::fmt::Result {
         let arch_str = match self {
-            Architecture::Aarch32 => "aarch32",
-            Architecture::Aarch64 => "aarch64",
+            Architecture::Aarch32 => "arm",
+            Architecture::Aarch64 => "arm64",
```

Comment:
Looking at rust and zig targets, I think this should be changed to `aarch64` to be consistent with those toolchains. 

This will be a breaking change for platforms that have prebuilt binaries.

Reply:
This is confusing me...

> I think this should be changed to aarch64 to be consistent with those toolchains.

Why is it changed to arm now?